src/components/auth/SignInScreen.js

```diff
@@ -103,104 +103,138 @@ export default class SignInScreen extends React.Component {
                       onChangeText={value => this.onChangeText('password', value)}
                     />
                   </Item>
+                  {/* Sign Up Text */}
+                  <Text style={styles.buttonText2}>Novo usuário?{" "}
+                    <Text 
+                      onPress={() => this.props.navigation.navigate('SignUp')}
+                      style={styles.buttonText2link}>Crie uma conta 
+                    </Text>
+                    {" "}»
+                  </Text>
+                  {/* Forgot Password Text */}
+                  <Text style={styles.buttonText2}>Esqueceu sua senha?{" "}
+                    <Text
+                      onPress={() => this.props.navigation.navigate('ForgotPassword')}
+                      style={styles.buttonText2link}>Recupere aqui
+                    </Text>
+                    {" "}»
+                  </Text>
+                  {/* Loading ActivityIndicator */}
+                </View>  
+                <KeyboardAvoidingView style={styles.container2} behavior='position' enabled>
                   {/* Sign In Button */}
                   <TouchableOpacity
                     onPress={() => this.signIn()}
```

Comment:
Please add empty string checks for email and password inside of signIn()

Reply:
i made it outside of sign in if that is okay like the sign up screen

Reply:
yea thats fine